hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetOrgUnitTypesOptions.ts

```diff
@@ -1,26 +1,43 @@
 import { getRequest } from '../../../../libs/Api';
 import { useSnackQuery } from '../../../../libs/apiHooks';
+import { memoize } from 'lodash';
+import { UseQueryResult } from 'react-query';
 
-export const useGetOrgUnitTypesOptions = () => {
+type DropDownOption = {
+    value: number;
+    label: string;
+    orignal: any;
+};
+const apiToDropDown = data => {
+    if (!data) return [];
+    return data.map(orgUnitType => ({
+        value: orgUnitType.id,
+        label: orgUnitType.name,
+        original: orgUnitType,
+    }));
+};
+
+export const useGetOrgUnitTypesOptions: (
+    number?,
+) => UseQueryResult<DropDownOption[]> = rootOrgUnitId => {
     const params = {
-        all: 'true',
-        order: 'name',
-        fields: 'name,id,depth',
+        org_unit_id: rootOrgUnitId,
+        version_id: ':default',
     };
-    const queryString = new URLSearchParams(params);
+    const filteredParams = Object.entries(params).filter(
+        // eslint-disable-next-line no-unused-vars
+        ([_key, value]) => value !== undefined,
```

Comment:
if you use only `_`, no need to disable eslint

Reply:
eslint still make me an error if I don't disable it, am I missing a config?